middle_end/flambda2/to_jsir/to_jsir.ml

```diff
@@ -1,3 +1,205 @@
-let unit ~offsets:_ ~all_code:_ ~reachable_names:_ flambda_unit =
-  ignore flambda_unit;
+open! Flambda2_terms.Flambda.Import
+
+(** Bind a fresh JSIR variable to [expr], and map [fvar] to this new variable in the
+    environment. *)
+let bind_expr_to_var ~env ~res fvar expr =
+  let jvar = Jsir.Var.fresh () in
+  ( To_jsir_env.add_var env fvar jvar,
+    To_jsir_result.add_instr res (Jsir.Let (jvar, expr)) )
+
+(** Bind a fresh JSIR variable to [expr], and map [fvar] to this new variable in the
+    environment. *)
+let bind_expr_to_symbol ~env ~res symbol expr =
+  let jvar = Jsir.Var.fresh () in
+  ( To_jsir_env.add_symbol env symbol jvar,
+    To_jsir_result.add_instr res (Jsir.Let (jvar, expr)) )
+
+(** Bind a fresh variable to the result of translating [simple] into JSIR, and
+    map [fvar] to this new variable in the environment. *)
+let create_let_simple ~env ~res fvar simple =
+  Flambda2_term_basics.Simple.pattern_match' simple
```

Comment:
I think you should just be able to use `Simple.` here if you replace the `open` at the top with `open! Flambda.Import`.